beta/aws-ecsfargate-cfn/README.md

```diff
@@ -0,0 +1,220 @@
+
+*Example command:*
+
+```sh
+aws cloudformation deploy \
+    --template-file ./op-scim-bridge.yaml \
+    --stack-name op-scim-bridge \
+    --capabilities CAPABILITY_IAM \
+    --parameter-overrides \
+        Route53HostedZoneID=Z2ABCDEF123456 \
+        DomainName=scim.example.com \
+        scimsession=$(cat ./scimsession) \
+        # VPCCIDR=10.0.0.0/16 \
+    # --tags \
+    #     Key1=Value1 \
+    #     Key2=Value2
+```
+
+1. If the template file was saved with a different file name or somewhere other than the working directory, replace `./op-scim-bridge.yaml` with the actual path (for example, `--template-file path/to/template.file`).
+2. If desired, set your own value for the `--stack-name` flag in the above command to choose your own name (for example, `--stack-name your-stack-name`).
+   > **Note**
+   >
+   > 📄 We use `op-scim-bridge` as a shorthand for 1Password SCIM Bridge throughout our examples and supporting documentation, but the choice is arbitrary. CloudFormation will use the stack name (or a truncated version where needed) as a prefix when naming the created AWS resources.
+3. Leave the default value for the VPC CIDR range (`VPCCIDR`). If you require something different for your environment, you can uncomment this line and replace the CIDR range with your own value.
+4. Replace `Z2ABCDEF123456` above with the ID of the Route 53 hosted zone in which to create DNS records.
+5. Replace `scim.example.com` with a domain name for your SCIM bridge that is in the domain of this hosted zone.
+6. Ensure your `scimsession` file is located in the working directory, or replace the path specified in the subshell command (for example, `scimsession=$(cat path/to/scimsession.filename)`).
+   > **Note**
+   >
+   > 💻 If you saved your `scimsession` file as an item in your 1Password account, you can [use 1Password CLI to pass this value](https://developer.1password.com/docs/cli/secrets-scripts#option-2-use-op-read-to-read-secrets) (for example, `scimsession=$(op read "op://Private/scimsession file/scimsession")`)
+7. Optionally, uncomment the appropriate lines and add tags as key-value pairs to apply to all supported resources in the stack.
+
+You should see:
+
+> ```sh
+> Waiting for changeset to be created..
+> Waiting for stack create/update to complete
+> ```
+
+More detailed information is available in the CloudFormation console. A success message will be returned when the stack is created, for example:
+
+> ```sh
+> Successfully created/updated stack - op-scim-bridge
+> ```
+
+## Test your SCIM bridge
+
+A clickable link of the URL for your SCIM bridge is available in the Outputs tab of the CloudFormation console (for example, <https://scim.example.com/>). You can sign in to this URL with the bearer token for your SCIM bridge to test, view status information, or retrieve logs. You can also send an authenticated SCIM API request to your SCIM bridge from the command line.
```

Comment:
Any reason why `<>` are wrapping the URL here? I don't think it's necessary for readability personally, and I'd even do a inline code highlight if anything:

```suggestion
A clickable link of the URL for your SCIM bridge is available in the Outputs tab of the CloudFormation console (for example, `https://scim.example.com/`). You can sign in to this URL with the bearer token for your SCIM bridge to test, view status information, or retrieve logs. You can also send an authenticated SCIM API request to your SCIM bridge from the command line.
```